legion/engine/core/core.vcxproj

```diff
@@ -85,7 +85,8 @@ copy /Y "$(SolutionDir)xcopyexclude" ".\"
 
 xcopy "$(ProjectDir)..\$(ProjectName)" "$(SolutionDir)include\$(ProjectName)\" /i /s /r /exclude:xcopyexclude /y &gt; nul
 
-del /q ".\xcopyexclude"</Command>
+del /q ".\xcopyexclude" 
+$(SolutionDir)tools\RythePreprocessor\RytheTributary.exe -root=D:\Repos\Rythe-legacy\ -sysinclude=deps\include -include=legion\engine\ -module=legion\engine\core -module=legion\engine\rendering -module=legion\engine\application -module=legion\engine\audio -module=applications\sandbox -ex=deps -ex=glm -ex=autogen</Command>

```

Comment:
core shouldn't autogen for other modules, and the root does not work for every computer